spec/javascripts/modules/coronavirus-local-restrictions-postcode-form_spec.js

```diff
@@ -0,0 +1,50 @@
+describe('CoronavirusLocalRestrictionsPostcodeForm', function () {
+  'use strict'
+
+  var input, button, container
+
+  beforeEach(function () {
+    container = document.createElement('div')
+    container.innerHTML =
+      '<form data-module="coronavirus-local-restrictions-postcode-form">' +
+        '<input name="postcode">' +
+        '<button type="submit">Submit</button>' +
+      '</form>'
+    document.body.appendChild(container)
+
+    var form = container.querySelector('form')
+    form.addEventListener('submit', function (e) {
+      e.preventDefault()
+    })
+
+    input = form.querySelector('input')
+    button = form.querySelector('button')
+
+    var postcodeForm = new GOVUK.Modules.CoronavirusLocalRestrictionsPostcodeForm()
+    postcodeForm.start($(form))
```

Comment:
```suggestion
    postcodeForm.start(form)
```

We stopped using jQuery in our JS now - does this still work as expected

Reply:
It would work, but it doesn't feel very realistic to fake it as a different type of object when we can use the actual type.

Reply:
We've started doing it in a few places now - https://github.com/alphagov/collections/blob/master/spec/javascripts/modules/coronavirus-track-local-restrictions_spec.js#L25

I'm happy either way but seeing as you went through the trouble of adding vanilla JS everywhere - it would be good to just not use it completedly.

Reply:
Sure, I feel like when we have a jQuery object in use it makes sense to use it, otherwise your tests are less realistic while not actually bringing us closer to removing jQuery - given that we'll need to edit the interface of the `GOVUK.Modules.start` pattern to remove jQuery.